modules/collection/programs/hyprpanel.nix

```
@@ -35,22 +36,20 @@ in {
       };
       description = ''
         JSON-style configuration for HyprPanel, written to
-        {file}`$HOME/.config/hyprpanel/hyprpanel.conf`.
-         Refer to [HyprPanel documentation](https://hyprpanel.com/getting_started/installation.html).
+        {file}`$HOME/.config/hyprpanel/config.json`.
+         Refer to [HyprPanel documentation](https://hyprpanel.com/configuration/settings.html).
       '';
     };
   };
 
   config = mkIf cfg.enable {
     packages = [cfg.package];
 
-    # Integration with Hyprland exec-once
-    hj.rum.programs.hyprland.settings.exec-once = mkIf (cfg.hyprland.enable && config.hj.rum.programs.hyprland.enable) [
-      "${cfg.package}/bin/hyprpanel"
-    ];
+    # Integration with Hyprland exec-once 
+    rum.programs.hyprland.settings.exec-once = mkIf cfg.hyprland.enable [ lib.getExe cfg.package ];
```

Since you added the inherit, the `lib.` prefix can be removed
```suggestion
    rum.programs.hyprland.settings.exec-once = mkIf cfg.hyprland.enable [ getExe cfg.package ];
```